pkgvars

```diff
@@ -1,6 +1,6 @@
 pkgname=citus-enterprise
 pkgdesc='Citus Enterprise'
-pkglatest=9.5.7.citus-1
+pkglatest=10.0.5.citus-1
 releasepg=11,12,13
 nightlypg=11,12,13
```

Comment:
Would you like to have a similar change like that in https://github.com/citusdata/packaging/pull/723/files#diff-eec17024f883b01dfe5724f2a923a4d476504b83d4bc5150a1db0b1eca1777a7L5 ?